src/python/gem5/resources/downloader.py

```diff
@@ -215,6 +227,7 @@ def get_resource(
     clients: Optional[List] = None,
     gem5_version: Optional[str] = core.gem5Version,
     quiet: bool = False,
+    proxy_context: Optional[object] = None,
```

Comment:
I'm pretty sure the type is https://docs.python.org/3/library/ssl.html#ssl.SSLContext